easybuild/easyconfigs/b/BBMap/BBMap-39.14-GCC-12.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '39.14'
```

Comment:
latest version is `39.19` -- is there a particular reason to stick to an older version?